website/releases/release-0.14.0.md

```diff
@@ -0,0 +1,339 @@
+For example, `parquet.bloom.filter.enabled#rider=true` creates a bloom filter for the rider column. Whenever a query 
+involves a predicate on the rider column, the bloom filter comes into play, enhancing read performance.
+
+#### Incremental queries with multi-writers
+In multi-writer scenarios, there can be instances of gaps in the timeline (requests or inflight instants that are not 
+the latest instant) due to concurrent writing activities. These gaps may result in inconsistent outcomes when 
+performing incremental queries. To address this issue, Hudi 0.14.0 introduces a new configuration setting, 
+[`hoodie.read.timeline.holes.resolution.policy`](https://hudi.apache.org/docs/configurations#hoodiereadtimelineholesresolutionpolicy), 
+specifically designed for handling these inconsistencies in incremental queries. The configuration provides three possible policies:
+- `FAIL`: This serves as the default policy and throws an exception when such timeline gaps are identified during an incremental query.
+- `BLOCK`: In this policy, the results of an incremental query are limited to the time range between the holes in the 
+   timeline. For instance, if a gap is detected at instant t1 within the incremental query range from t0 to t2, the 
+   query will only display results between t0 and t1 without failing.
+- `USE_TRANSITION_TIME`: This policy is experimental and involves using the state transition time, which is based on the 
+   file modification time of commit metadata files in the timeline, during the incremental query.
+
+#### Timestamp support with Hive 3.x
+For quite some time, Hudi users encountered [challenges](https://issues.apache.org/jira/browse/HUDI-83) regarding reading Timestamp type columns written by Spark and 
+subsequently attempting to read them with Hive 3.x. While in Hudi 0.13.x, we introduced a 
+[workaround](https://github.com/apache/hudi/commit/cd314b8cfa58c32f731f7da2aa6377a09df4c6f9#diff-cff4dfc264f7abcac63a5ba5db55b38115177fe279ab35807d345c2b8872475e)
+to mitigate this issue, version 0.14.0 now ensures full compatibility of HiveAvroSerializer with Hive 3.x to resolve this.
+
+#### Google BigQuery sync enhancements
+With 0.14.0, the [BigQuerySyncTool](https://hudi.apache.org/docs/gcp_bigquery) supports syncing table to BigQuery 
+using [manifests](https://cloud.google.com/blog/products/data-analytics/bigquery-manifest-file-support-for-open-table-format-queries).
+This is expected to have better query performance compared to legacy way. Schema evolution is supported with the manifest approach. 
+Partition column no longer needs to be dropped from the files due to new schema handling improvements. To enable this 
+feature, users can set 
+[`hoodie.gcp.bigquery.sync.use_bq_manifest_file`](https://hudi.apache.org/docs/configurations#hoodiegcpbigquerysyncuse_bq_manifest_file) 
+to true.
+
+### Spark read side improvements
+
```

Comment:
In the spark query side improvement, should we talk about RLI benefits for keyed lookups and deleted?